src/components/NewWorkspaceModal.js

```diff
@@ -175,6 +195,30 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           }), billingProjects)).sort()
         })
       ])]),
+      h(IdContainer, [id => h(Fragment, [
+        h(FormLabel, { htmlFor: id }, [
+          'Bucket location',
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
+            'A bucket location can only be set when creating a workspace. ',
+            'Once set, it cannot be changed. ',
+            'A cloned workspace will automatically inherit the bucket location from the original workspace but this may be changed at clone time.',
+            p([
+              'By default, workflow and Cloud Environment VMs will run in the same region as the workspace bucket. ',
+              'Changing bucket or VM locations from the defaults can lead to network egress charges.'
+            ]),
+            h(Link, {
+              href: 'https://support.terra.bio/hc/en-us/articles/360058964552',
+              ...Utils.newTabLinkProps
+            }, ['Read more about bucket locations'])
+          ])
+        ]),
+        h(Select, {
+          id,
+          value: bucketLocation,
+          onChange: ({ value }) => setBucketLocation(value),
```

Comment:
`value` is a string, correct? If so, we wouldn't want to do object deconstruction (i.e. ` { value }`)?
Also,
```javascript
onChange: value => setBucketLocation(value),
```

can be written as
```javascript
onChange: setBucketLocation,
```